include/deal.II/dofs/dof_renumbering.h

```diff
@@ -657,6 +656,14 @@ namespace DoFRenumbering
   void
   block_wise (DoFHandler<dim,spacedim> &dof_handler);
 
+  /**
+   * Sort the degrees of freedom by vector block. It does the same thing as the
+   * above function, only that it does this for one single level of a multilevel
+   * discretization. The non-multigrid part of the the DoFHandler is not touched.
+   */
+  template <int dim, int spacedim>
+  void
+  block_wise (DoFHandler<dim,spacedim> &dof_handler, const unsigned int level);
```

Comment:
Correct, it's unnecessary from a language point of view to declare the parameter `const`. We use it as part of the documentation to make it plain and obvious to the reader that this is an input parameter (that is implicitly already clear because the argument is passed by value, but having a convention obviates the need to think about such things).